### PR TITLE
[master] [cc] Mention about K8 deployment option for production environments

### PR DESCRIPTION
**Pull request by slahirucd7**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

## Samples
image

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Let's start Choreo Connect with a WSO2 API Manager instance which will be acting
     - Minimum CPU : 4vCPU
     - Minimum Memory : 4GB
 
+!!! note
+    Choreo Connect Docker based deployment option explained here **only for the tryout purpose.** This method is **not recommended to use in production deployments**. For production deployments you can use Kubernetes based Choreo Connect deployment with [Helm artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts/) or [YAML artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane/).
```
Comment (suksw) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim.md:13]:
```suggestion
    The Choreo Connect Docker Compose based deployment option explained here is **only for tryout purposes.** This method is **not recommended for production deployments**. For production deployments, you can use the Kubernetes based Choreo Connect deployment with [Helm artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts/) or [YAML artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane/).
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ You can deploy Choreo Connect using one of the following options.
 - [API Manager as Control Plane]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/)
 - [Standalone Gateway]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway/)
 
+!!! note
+    Choreo Connect Docker based deployment option provided **only for the tryout purpose.** Docker based deployment option is **not recommended to use in production deployments**. For production deployments you can use Kubernetes based Choreo Connect deployment with [Helm artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts/) or [YAML artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane/).
```
Comment (suksw) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md:11]:
```suggestion
    The Choreo Connect Docker Compose based deployment option explained here is **only for tryout purposes.** This method is **not recommended for production deployments**. For production deployments, you can use the Kubernetes based Choreo Connect deployment with [Helm artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts/) or [YAML artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane/).
```

Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md:11]: Kubernetes should be used for Choreo Connect production deployments. The common guidelines for deploying Choreo Connect in a **production environment** using the [Helm deployment option](base path/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-deploy-overview/) are provided below.

Comment (pubudu538) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md:11]: This is not required.

**Review by pubudu538 — APPROVED** [review]

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**